### PR TITLE
Dynamically determine package location

### LINKED ISSUE #13: Script not properly searching for "MainActivity.java"
Hi,

the `set-splash` command doesn't work if the folder structure leading to `MainActivity.java` is not the default one / has been changed. In my case, the **actual** path of MainActivity.java is:

```
./android/app/src/main/java/com/nerdybirdy/boilerplate/MainActivity.java
                            com.nerdybirdy.boilerplate <- package name
```
This is because I renamed my a app using <a href="github.com/junedomingo/react-native-rename">react-native-rename</a>, which inserts the "extra folder" when changing the bundle name to "com.**nerdybirdy**.boilerplate"

The script is searching for the file in a directory based on the "name" value of the `package.json`, which leads to it failing to find the file:
```
./android/app/src/main/java/com/nbboilerplate/MainActivity.java
                            com.nbboilerplate <- not the package name, but "com" + packageJson.name
```

I guess a solution would be to control/override the path leading to `MainActivity.java` with an extra command-line parameter. Would that be possible to add to your super helpful toolset? :)

Thanks!

### LINKED ISSUE #18: Feature Request: Add command line parameter to configure package name
By default, the command is using the `name` field from `package.json` to search for `MainActivity.java` and `AppDelegate.m`.  In our project, the path to these files is different from what is in package.json.  Can you add an option to configure this from the command line?

### LINKED ISSUE #24: Fetch package names from the platform files
As mentioned in #18 , the package name is not necessarily the platform packages names, but instead of passing in the values through cli, it is much more feasible to fetch the package names from the platform files, and therefore avoid another unnecessary step. This is the same method used by `react-native` in the linking process.
...
I am working on a PR for this, and should have it available in a day or two, after fully testing.

### PR DESCRIPTION
**Pull request by adibas03**
closes #13 
closes #18 
closes #22 
closes #24

### CODE REVIEW

**Comment by adibas03** [comment]
@yleflour 
Implementation is ready
Kindly review this

File: src/utils.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+//Resort to read file content to fetch package value rather than adding package to parse xml/podfile
+const ANDROID_PACKAGE_EXP = new RegExp(/package=\"(.+)\"/);
+const IOS_PACKAGE_EXP = new RegExp(/target\s*["'](.+)['"]\s*do/);
+
+function fetchFileLocation(filePath: string): string {
+  return join(process.cwd(), filePath);
+}
+
+function getPackageFileName(platform: string): string {
```
Comment (yleflour) [inline src/utils.ts]: should split into a `getAndroidPackageFileName` and `getIosPackageFileName`

Comment (yleflour) [reply src/utils.ts]: Just a small challenge, PR is too useful for this to prevent merging

Comment (adibas03) [reply src/utils.ts]: @yleflour  Sorry, just seeing this. Been in the cave.
Do I need to do anything on this, or can it be ignored till future updates?

**Pull request merged**